src/Compressors/Compression.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Spatie\DbDumper\Compressors;
+
+interface Compression
+{
+    /**
+     * Returns the command needed to compress.
+     *
+     * @returns string
+     */
+    public function getCommand();
```

Comment:
Remove the docblock, add a `string` return type

Reply:
Done